job-service-container/pom.xml

```diff
@@ -420,6 +427,15 @@
                                                 <unpack>true</unpack>
                                                 <useProjectArtifact>false</useProjectArtifact>
                                             </dependencySet>
+                                            <!-- Including the healthcheck web application. -->
+                                            <dependencySet>
+                                                <outputDirectory>adminapps/ROOT/</outputDirectory>
```

Comment:
A very minor point but I notice the output directories for the existing webapps don't end with a slash.  I assume it's not important but maybe we should be consistent.